style.css

```diff
@@ -1,15 +1,34 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+}
+main {
+  max-width: 350px;
+  margin-left: auto;
+  margin-right: auto;
+}
 h1 {
     font-size: 100px;
 }
-
-a img {
-    border:none;
-    outline:none;
+h2 {
+    text-align: center;
+}
+img {
+    max-width: 100%;
+}
+.social-icon {
+    text-decoration: none;
+}
+.social-icon {
```

Comment:
I believe my issue arises somewhere around here. I actually was having problems with the styling where the icons still wouldn't all fit on the same line on a mobile (iPhone 5) screen and realized I had to change some of the padding on the boxes to get them all to fit.

Now, the issue I'm having is that the social-icons seem to naturally align to the left. You can easily notice it on my page. How do we go about centering them? I'm imagining something with margins and padding, though my attempts at this have not proved fruitful.